openfe/setup/ligand_network_planning.py

```diff
@@ -303,3 +304,86 @@ def generate_network_from_indices(
         edges.append(mapping)
 
     return LigandNetwork(edges=edges, nodes=ligands)
+
+
+def load_nes_network(
+        ligands: list[SmallMoleculeComponent],
+        mapper: AtomMapper,
+        network_file: Union[str, Path],
+) -> LigandNetwork:
+    """Generate a LigandNetwork from an Orion NES network file.
+
+    Parameters
+    ----------
+    ligands : list of SmallMoleculeComponent
+      the small molecules to place into the network
+    mapper: AtomMapper
+      the atom mapper to use to construct edges
+    network_file : str
+      path to NES network file.
+
+    Returns
+    -------
+    LigandNetwork
+
+    Raises
+    ------
+    KeyError
+      If an unexpected line format is encountered.
+    """
+    
+    with open(network_file, 'r') as f:
+        network_lines = [l.strip().split(' ') for l in f
+                         if not l.startswith('#')]
+
+    names = []
+    for entry in network_lines:
+        if len(entry) != 3 or entry[1] != ">>":
+            errmsg = ("line does not match expected name >> name format: "
+                      f"{entry}")
+            raise KeyError(errmsg)
+
+        names.append((entry[0], entry[2]))
+
+    return generate_network_from_names(ligands, mapper, names)
+
+
+def load_fepplus_network(
+        ligands: list[SmallMoleculeComponent],
+        mapper: AtomMapper,
+        network_file: Union[str, Path],
+) -> LigandNetwork:
+    """Generate a LigandNetwork from an FEP+ edges network file.
+
+    Parameters
+    ----------
+    ligands : list of SmallMoleculeComponent
+      the small molecules to place into the network
+    mapper: AtomMapper
+      the atom mapper to use to construct edges
+    network_file : str
+      path to edges network file.
+
+    Returns
+    -------
+    LigandNetwork
+
+    Raises
+    ------
+    KeyError
+      If an unexpected line format is encountered.
+    """
+
+    with open(network_file, 'r') as f:
+        network_lines = [l.split() for l in f.readlines()]
+
+    names = []
+    for entry in network_lines:
+        if len(entry) != 5 or entry[1] != '#' or entry[3] != '->':
+            errmsg = ("line does not match expected name >> name format: "
```

Comment:
maybe change errormsg as in the fep+ edge file the format is name -> name?

Reply:
Oops. I looks like I copy-pasted and forgot to change 😅